### PR TITLE
[CPDLP-2523] Investigate if there is a data and/or logic bug for EDT GET declarations calls

### PR DESCRIPTION
**Pull request by mooktakim**
### Context

- Ticket: https://dfedigital.atlassian.net/browse/CPDLP-2523

### Changes proposed in this pull request

* Updated controller and query service classes to use the collection when returning single participant declaration.

### CODE REVIEW

File: app/services/api/v3/participant_declarations_query.rb [hunk 1]
```diff
@@ -57,6 +57,13 @@ def participant_declarations_from(paginated_join)
         scope
       end
 
+      def find(id)
```
Comment (cwrw) [inline app/services/api/v3/participant_declarations_query.rb:60]: suggestion: could we name it maybe `participant_declaration` similar to other services singular methods?

File: spec/requests/api/v1/participant_declarations_spec.rb [hunk 2]
```diff
@@ -759,6 +759,63 @@
           expect(response.status).to eq 404
         end
       end
+
+      context "when querying a single old participant declaration owned by another provider" do
```
Comment (cwrw) [inline spec/requests/api/v1/participant_declarations_spec.rb:763]: could you add a spec for the voided endpoint (that it won't allow you to void a declaration from another provider) here and in other versions?

**Review by cwrw — COMMENTED** [review]
looking good, some minor comments

File: app/services/api/v3/participant_declarations_query.rb (hunk 1 above)
Comment (mooktakim) [reply app/services/api/v3/participant_declarations_query.rb:60]: Updated

File: spec/requests/api/v1/participant_declarations_spec.rb (hunk 2 above)
Comment (mooktakim) [reply spec/requests/api/v1/participant_declarations_spec.rb:763]: added tests

**Review by cwrw — APPROVED** [review]
looks good 👍

**Pull request merged**